moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeFeatureCollections.js

```diff
@@ -30,13 +30,11 @@ export const getAllComponentFeatures = (component) => {
   });
   // Flatten array of arrays containing features from each feature table
   const allComponentFeaturesFlat = allComponentFeatures.flat();
-
   // Make features valid GeoJSON by adding type and properties attributes
   return allComponentFeaturesFlat.map((feature) => {
-    feature.properties = {
-      // we need this property to tie individual map features back to their parent component
-      project_component_id: component.project_component_id,
-    };
+    feature.properties ??= {};
```

Comment:
Makes sure not to overwrite existing properties, which will be present on edited features.

I'm still not totally sure how this working before—it obviously was!—but here's the bit of code throwing me off:
https://github.com/cityofaustin/atd-moped/blob/724bb200f672e57f60c07c2f6ca4ffad3a01710b/moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeFeatureCollections.js#L85-L89

I'd like to spend more time tracing the data flow, but this patch definitely brings us back to the right state so I'm going to leave it.